config/routes.rb

```diff
@@ -22,7 +22,8 @@
 
   namespace :admin do
     root to: 'legal_aid_applications#index'
-    resources :legal_aid_applications, only: %i[index destroy] do
+    resources :legal_aid_applications, only: %i[index update destroy] do
+      patch :create_test_applications, on: :collection
```

Comment:
That probably should be a `POST` - a resource is not being amended, new ones are being created.